velox/exec/Window.cpp

```diff
@@ -493,7 +499,7 @@ void Window::callApplyForPartitionRows(
 
   for (auto i = 0; i < numFuncs; i++) {
     const auto& windowFrame = windowFrames_[i];
-
+    validFrames_[i].resizeFill(numRows);
```

Comment:
Shouldn't we update the SelectivityVector to set all rows to true here or at the start of computeValidFrames?

Reply:
SelectivityVector.resizeFill(numRows) has a default to set all bits to true, so I didn't specify a parameter here. But I'll add one and a comment to make it more clear.

Reply:
Thank you for clarifying. Would it make sense to move this call to computeValidFrames? That function assumes SelectivityVector has all rows set to true. Moving this code there would make the assumption a bit clearer.

Reply:
We called computeValidFrames(...) only for kPreceding/kFollowing frames, so this place in the code is more appropriate. I hope the comment helps now.